src/sql/remove-site-users.pl

```diff
@@ -32,6 +32,10 @@
 my $REALM_SQL = "delete from SAKAI_REALM_RL_GR where realm_key in (select realm_key from sakai_realm where realm_id like '/site/SITE-ID%');\n";
 ## make site unpublished
 my $SITE_SQL = "update SAKAI_SITE set published=0 where site_id = 'SITE-ID'; \n";
+
+## save to CPM_ACTION_LOG table for tracking purposes
+my $LOG_SQL = "insert into CPM_ACTION_LOG (ACTION_TIME, SITE_ID, ACTION_TAKEN) VALUES (SYSTIMESTAMP,'SITE-ID','DELETE_MEMBERS'); \n";
```

Comment:
The timestamp value should be consistent when we update the table. This uses SYSTIMESTAMP and READ_ONLY uses CURRENT_TIMESTAMP.  In practice this probably won't make a difference unless the user and the system are in different timezone.  That could happen anytime we use a cloud database.